src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -512,4 +649,51 @@ private String beschreibung(String strUrlFeed) {
 
     }
 
+    private boolean isAllTextsEmpty(String... aTexts)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this method into "ThemaLaden". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3398)